docs/ECLLanguageReference/ECLR_mods/RecordStructure.xml

```diff
@@ -625,7 +625,7 @@ END;</programlisting>
             DICTIONARY lookup returns no match.</para><para>2. When an
             out-of-range record is fetched using ds[n] (as in ds[5] when ds
             contains only 4 records). </para><para>3. In the default record
-            passed to TRANSFORM functions in LEFT ONLY or RIGHT ONLY JOINs
+            passed to TRANSFORM functions in LEFT OUTER or RIGHT OUTER JOINs
```

Comment:
Strictly speaking it is used as the default for LEFT ONLY, LEFT OUTER, RIGHT ONLY, RIGHT OUTER and FULL XX.  
Maybe best to say
"In the default records passed to TRANSFORM functions in non-INNER JOINS where there is no corresponding row."